lms/djangoapps/student_account/urls.py

```diff
@@ -1,12 +1,14 @@
 from django.conf import settings
 from django.conf.urls import patterns, url
+from .views import RecoverPasswordView
 
 urlpatterns = []
 
 if settings.FEATURES.get('ENABLE_COMBINED_LOGIN_REGISTRATION'):
     urlpatterns += patterns(
         'student_account.views',
         url(r'^password$', 'password_change_request_handler', name='password_change_request'),
+        url(r'^recover-password$', RecoverPasswordView.as_view({'post': 'post'}), name="recover_password_api"),
```

Comment:
Is the dict necessary? Doesn't it just default to `post` anyway?

Reply:
I would like to be explicit :)

Reply:
I understand, but there's no reason to be explicit about a default, it's unidiomatic/unpythonic, or however one would like to call it. If you do a `git grep` for this:

```bash
$ git grep 'as_view({'
lms/djangoapps/discussion_api/api.py:    user_profile_details = AccountViewSet.as_view({'get': 'list'})(request).data
lms/djangoapps/notifier_api/tests.py:        self.list_view = NotifierUsersViewSet.as_view({"get": "list"})
lms/djangoapps/notifier_api/tests.py:        self.detail_view = NotifierUsersViewSet.as_view({"get": "retrieve"})
openedx/core/djangoapps/user_api/urls.py:ME = AccountViewSet.as_view({
openedx/core/djangoapps/user_api/urls.py:ACCOUNT_LIST = AccountViewSet.as_view({
openedx/core/djangoapps/user_api/urls.py:ACCOUNT_DETAIL = AccountViewSet.as_view({
```

The only one that uses the same value as the key:

```python
ME = AccountViewSet.as_view({
    'get': 'get',
})

ACCOUNT_LIST = AccountViewSet.as_view({
    'get': 'list',
})

ACCOUNT_DETAIL = AccountViewSet.as_view({
    'get': 'retrieve',
    'patch': 'partial_update',
})
```

And as you can see, that's because it's *needed*.